custom_components/wienerlinien/sensor.py

```diff
@@ -42,11 +42,12 @@ async def async_setup_platform(hass, config, add_devices_callback, discovery_inf
     for stopid in stops:
         api = WienerlinienAPI(async_create_clientsession(hass), hass.loop, stopid)
         data = await api.get_json()
-        try:
-            name = data["data"]["monitors"][0]["locationStop"]["properties"]["title"]
-        except Exception:
-            raise PlatformNotReady()
-        dev.append(WienerlinienSensor(api, name, firstnext))
+        for monitor in data["data"]["monitors"]:
+            try:
+                name = monitor["locationStop"]["properties"]["title"] + " - "+ monitor["lines"][0]["name"]
```

Comment:
This is a breaking change

Reply:
ok np. I'll just use it for myself then.